.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: java
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk7
```

Comment:
Does this config mean the entire build will be tested 3 times?  Does this change make the Travis CI tests slower?